plugins/GSdx/GSRenderer.h

```diff
@@ -49,6 +49,7 @@ class GSRenderer : public GSState
 	bool m_fxaa;
 	bool m_shadeboost;
 	bool m_texture_shuffle;
+	GSVector2i real_size;
```

Comment:
`m_real_size`? You should probably initialise it as well, otherwise coverity will complain.